app/views/shared/views/ems_common/angular/_form.html.haml

```diff
@@ -81,10 +81,13 @@
                             "ng-model"                => "emsCommonModel.azure_tenant_id",
                             "required"                => "",
                             "checkchange"             => "",
+                            "ng-pattern"              => "/^[A-Fa-f0-9]{8}-[A-Fa-f0-9]{4}-[A-Fa-f0-9]{4}-[A-Fa-f0-9]{4}-[A-Fa-f0-9]{12}$/",
                             "prefix"                  => "default",
                             "reset-validation-status" => "default_auth_status"}
         %span.help-block{"ng-show" => "angularForm.azure_tenant_id.$error.required"}
           = _("Required")
+        %span.help-block{"ng-show" => "angularForm.azure_tenant_id.$error.pattern"}
+          = _("Invalid GUID")
```

Comment:
I'm not entirely sure the "Invalid GUID" message here will not be a bit confusing (I'm putting in a tenant id,
not a guid; what's guid? etc.)

@dclarizio What do you think?